src/core/Akka.Cluster/ClusterHeartbeat.cs

```diff
@@ -63,7 +63,7 @@ public static Props Props(Cluster getCluster)
     /// <summary>
     /// INTERNAL API
     /// </summary>
-    internal class ClusterHeartbeatSender : ReceiveActor
+    internal class ClusterHeartbeatSender : ReceiveActor, IWithTimers
```

Comment:
This class is inherited for testing purposes, no sealing. But did add `IWithTimers`.